docs/selectively-triggering-aktualizr.adoc

```diff
@@ -2,7 +2,7 @@
 
 Aktualizr normally runs as a `systemd` service and regularly polls for updates. The default polling interval is set at 10s, and this should be increased by a few orders of magnitude in a production system.
 
-It is also possible to trigger the update cycle, or individual parts of the update cycle, manually or programmatically. This is done by using aktualizr's `--running-mode` option (or by setting `running_mode` in the `[uptane]` configuration section).
+It is also possible to trigger the update cycle, or individual parts of the update cycle, manually or programmatically. This is done by using aktualizr's `--running-mode` option, or just supplying the operation name as a subcommand.
```

Comment:
You still can use the configuration option for now, but I'd like to remove it soon, so it's fine with me to remove this.

Reply:
Yes, as we still don't know why would one use it, I thought of removing it from the docs first.

Reply:
Well, I didn't really look for that in particular, I was only documenting the "subcommand" syntax. It was convenient to just ditch it as well for clarity.